src/core/lib/address_utils/sockaddr_utils.cc

```diff
@@ -292,6 +330,8 @@ const char* grpc_sockaddr_get_uri_scheme(
       return "ipv6";
     case GRPC_AF_UNIX:
       return "unix";
+    case GRPC_AF_VSOCK:
```

Comment:
I believe this should be guarded by `#ifdef GRPC_HAVE_VSOCK`

Reply:
done